test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
+  test("returns a reversed array of the number's digits", () => {
+    expect(numberToReversedDigits(12)).toEqual([2, 1]);
```

Comment:
Could definitely use a few more test cases here to be sure the function does what its supposed to do in different situations